### ISSUE TITLE
A11y: Placeholder text for input fields have insufficient contrast

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5024]
Placeholders in input fields in the checkout should have a contrast of at least 4.5:1

!image-20241008-111219.png|width=1974,height=959,alt="image-20241008-111219.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-38784**



[created from NEXT-38784, comment 501649]
